packages/cli/test/integration.test.ts

```diff
@@ -3683,3 +3683,41 @@ test('vercel.json configuration overrides in an existing project do not prompt u
   text = await page.text();
   expect(text).toMatch(/Next\.js Test/);
 });
+
+test('[vc build] should use --local-config over default vercel.json', async () => {
```

Comment:
It'd prefer to add this as a unit test instead.

Reply:
Fixed in 8fac37560bfadce326a36bed1563da3196350856